config/manager/manager.yaml

```diff
@@ -65,15 +65,12 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-            - name : SOP_ENDPOINT
-              value: test.url
-            - name : ALERT_SMTP_FROM_ADDR
-              value: test@redhat.com
+            - name: SOP_ENDPOINT
+            - name: ALERT_SMTP_FROM_ADDR
             - name: ADDON_NAME
-              value: mcg-osd
             - name: NOOBAA_CORE_IMAGE
-              value: registry.redhat.io/odf4/mcg-core-rhel8@sha256:4ff2d65ea16dd1026fe278a0f8ca920f300dfcee205b4b8ede0ab28be1aa43a6
             - name: NOOBAA_DB_IMAGE
-              value: registry.redhat.io/rhel8/postgresql-12@sha256:be7212e938d1ef314a75aca070c28b6433cd0346704d0d3523c8ef403ff0c69e
+            - name: RHOBS_ENDPOINT
+            - name: RH_SSO_TOKEN_ENDPOINT
```

Comment:
Can we remove these keys, since they are not needed here?

Reply:
I think that this is a good place to have the whole list of env. vars. If i remove them from here, probably it will be more hard to understand what are( and to set) the "env vars requeriments" for developer environments. 
I think that this does not cause any harm. Am i wrong?

Reply:
It may de-initialize the vars. Can you comment them out? That would satisfy your use case.
(and mine too)

Reply:
I prefer to remove them.